src/Cron_Command.php

```diff
@@ -0,0 +1,234 @@
+	 *    <number>us - microseconds
+	 *    <number>ns - nanoseconds
+	 *
+	 *    So 1h10m2s is also a valid format
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Adds a cron job on example.com every 10 minutes
+	 *     $ ee cron add example.com --command='wp cron event run --due-now' --schedule='@every 10m'
+	 *
+	 *     # Adds a cron job to host running EasyEngine
+	 *     $ ee cron add host --command='wp cron event run --due-now' --schedule='@every 10m'
+	 *
+	 *     # Adds a cron job to host running EasyEngine
+	 *     $ ee cron add host --command='wp cron event run --due-now' --schedule='@every 10m'
+	 *
+	 *     # Adds a cron job to host running EasyEngine
+	 *     $ ee cron add host --command='wp cron event run --due-now' --schedule='@every 10m'
+	 *
+	 */
+	public function add( $args, $assoc_args ) {
+		EE\Utils\delem_log( 'ee cron add start' );
+
+		if ( ! isset( $args[0] ) || $args[0] !== 'host' ) {
+			$args = EE\Utils\set_site_arg( $args, 'cron' );
+		}
+
+		$site     = EE\Utils\remove_trailing_slash( $args[0] );
+		$command  = EE\Utils\get_flag_value( $assoc_args, 'command' );
+		$schedule = EE\Utils\get_flag_value( $assoc_args, 'schedule' );
+
+		if ( '@' !== substr( trim( $schedule ), 0, 1 ) ) {
+			$schedule_length = strlen( explode( ' ', trim( $schedule ) ) );
+			if ( $schedule_length <= 5 ) {
+				$schedule = '0 ' . trim( $schedule );
+			}
+		}
+
+		EE::db()->insert([
+			'site' => $site,
+			'command' => $command,
+			'schedule' => $schedule
+		], 'cron' );
+
+		$this->update_cron_config();
+
+		EE\Utils\delem_log( 'ee cron add end' );
+	}
+
+	/**
+	 * Lists scheduled cron jobs.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<site-name>]
+	 * : Name of site to whose cron will be displayed.
+	 *
+	 * [--all]
+	 * : View all cron jobs.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Lists all scheduled cron jobs
+	 *     $ ee cron list
+	 *
+	 *     # Lists all scheduled cron jobs of a site
+	 *     $ ee cron list example.com
+	 * @subcommand list
+	 */
+	public function _list( $args, $assoc_args ) {
+		$where = [];
+		$all   = EE\Utils\get_flag_value( $assoc_args, 'all' );
+
+		if ( ( ! isset( $args[0] ) || $args[0] !== 'host' ) && ! $all ) {
+			$args = EE\Utils\set_site_arg( $args, 'cron' );
+		}
+
+		if ( isset( $args[0] ) ) {
+			$where = [ 'site' => $args[0] ];
+		}
+
+		$crons = EE::db()->select( [], $where, 'cron' );
+
+		if ( false === $crons ) {
+			EE::error( 'No cron jobs found.' );
+		}
+
+		EE\Utils\format_items( 'table', $crons, [ 'id', 'site', 'command', 'schedule' ] );
+	}
+
+
+	/**
+	 * Generates cron config from DB
+	 */
+	private function update_cron_config() {
+
+		$config = $this->generate_cron_config();
+
+		file_put_contents( EE_CONF_ROOT . '/cron/config.ini', $config );
+		EE_DOCKER::restart_container( 'ee-cron-scheduler' );
+	}
+
+	/**
+	 * Generates and returns cron config from DB
+	 */
+	private function generate_cron_config() {
+		$config_template = file_get_contents( __DIR__ . '/../templates/config.ini.mustache' );
```

Comment:
Please test this path if it is getting included in the phar.

Reply:
@mbtamuli Tested. Works like charm.